ibc/types.go

```diff
@@ -62,6 +63,14 @@ func (c ChainConfig) MergeChainSpecConfig(other ChainConfig) ChainConfig {
 	return c
 }
 
+// GetShell returns the shell that should be used when executing scripts.
+func (c ChainConfig) GetShell() string {
+	if c.Shell == "" {
+		return "sh"
+	}
+	return c.Shell
+}
+
```

Comment:
For now, I think we should assume all containers have `sh` available. It will likely be a long time before we run into something we can't do with plain `sh`.